uiowa/uiowa
# [Image block] Added stretch to fit style

Resolves https://github.com/uiowa/uiowa/issues/6803

# How to test

```
ddev yarn workspace uids_base build && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/1776/layout
```

Add "Stretch image to vertically fill container" style to the image block on the page and verify that the image fills the entire container.
